### ISSUE TITLE
Whey build system

### ISSUE THREAD

**Issue by TyberiusPrime** [nix-community/poetry2nix#1723]
There's a small number of packages (e.g. dict2css) that appear to use a build system called 'whey'.
Adding that as a build override doesn't work, nix doesn't have a whey package in the right place.
Adding it to known-build-systems doesn't do anything.

I think from a cursory reading of the code that the build systems actually come from nixpkgs, is that correct?

**Comment by adisbladis** [comment]
> I think from a cursory reading of the code that the build systems actually come from nixpkgs, is that correct?

Correct. The way to go about this is to add whey to nixpkgs, only then will adding it to the known build systems actually work.

**Comment by TyberiusPrime** [comment]
I've written a nixpkgs PR https://github.com/NixOS/nixpkgs/pull/328025
